tests/run_tests.sh

```diff
@@ -113,6 +115,12 @@ function setup_test_options()
     else
         TEST_TOPOLOGY_OPTIONS="--topology ${TOPOLOGY}"
     fi
+
+    PYTEST_UTIL_OPTS=${PYTEST_COMMON_OPTS}
+    # Max failure only applicable to the test session. Not the preparation and cleanup session.
```

Comment:
@lguohan I can't remember off the top of my head - would a failure in `announce_routes` or `neighbor_health` trigger a failure in the old/current PR tests?

Reply:
This is a minor detail we can come back to later. I'll merge this PR so that you can work on your next change.